app/furniture/marketplace/menu/product_component.html.erb

```diff
@@ -1,16 +1,26 @@
 <%= render CardComponent.new(dom_id: dom_id(product)) do |card| %>
-  <%- card.with_header(variant: :no_padding) do %>
-    <% if product.photo.present? %>
+
+  <% if product.photo.present? %>
+    <%- card.with_header(variant: :no_padding) do %>
       <figure>
         <%= image_tag hero_image, class: "rounded-t-lg w-full" %>
         <figcaption class="px-2 pt-4 sm:px-4">
           <h3><%= name %></h3>
+          <p class="text-sm">
+            Serves <%= product.servings %>
+          </p>
         </figcaption>
       </figure>
-    <%- else %>
-      <h3 class="px-4"><%= name %></h3>
+    <%- end %>
+  <%- else %>
+    <%- card.with_header do %>
+      <h3><%= name %></h3>
+      <p class="text-sm">
+        Serves <%= product.servings %>
+      </p>
```

Comment:
These 4 lines are repeated 4 times, should pull them into a `Marketplace::Product::CardHeader`? or something?

Reply:
There's no component too small!

Reply:
I long for `<Marketplace::Product::CardHeader data-product="{{ product }}">` syntax 🙃

Reply:
Yeah, ViewComponent feels a little behind in this respect.